config/db/dev-gcp.yml

```diff
@@ -2,10 +2,10 @@ kafkaPool: nav-dev
 azure:
   enabled: true
 database:
-  - name: inntektsmelding
-    users: [ simba_datastream_bruker ]
-logical_decoding: true
-dbTier: db-custom-1-3840
+  name: inntektsmelding
+  tier: db-custom-1-3840
+  logicalDecoding: true
+  user: simba_datastream_bruker
```

Comment:
Vet du hvorfor vi må ha en user her, men ikke for notifikasjon og feilbehandler?

Reply:
Dataen fra denne dben streames til BigQuery, som trenger sin egen bruker. Det gjør vi ikke for de andre dbene.

Reply:
Gir mening 👍